src/sagemaker_xgboost_container/data_utils.py

```diff
@@ -25,8 +25,8 @@
 LIBSVM = 'libsvm'
 
 
-INVALID_CONTENT_TYPE_ERROR = "{invalid_content_type} is not an accepted ContentType:" \
-                             " 'csv', 'libsvm', 'text/csv', 'text/libsvm', 'text/x-libsvm'."
+INVALID_CONTENT_TYPE_ERROR = "{invalid_content_type} is not an accepted ContentType: " \
```

Comment:
Minor. I would've expected ```text/csv;label_size=1``` without space. Did you check that this is the conventional pattern?

Reply:
Yes, the public sagemaker documentation shows a space: https://docs.aws.amazon.com/sagemaker/latest/dg/cdf-training.html